content/setup.qmd

```diff
@@ -11,9 +11,21 @@ if (any(installed_packages == FALSE)) {
   install.packages(packages[!installed_packages], repos = "http://cran.us.r-project.org")
 }
 
-remotes::install_github("NOAA-FIMS/FIMS")
+# Install FIMS: main branch version if on main, dev version on any other branch
+
+branch_name <- system("git branch --show-current")
+use_fims_main <- grepl("main", branch_name)
+
+if (use_fims_main){
+  remotes::install_github("NOAA-FIMS/FIMS")
+} else {
+  remotes::install_github("NOAA-FIMS/FIMS", ref = "dev")
+}
+
+)
```

Comment:
I think that this line should be removed because I do not see the matching bracket.

Reply:
Yep! It made the render fail.